it/utils/IntegrationTest.scala

```diff
@@ -53,7 +54,7 @@ trait IntegrationTest extends AnyWordSpec with Matchers with GuiceOneServerPerSu
   val mtditid: String = anEmploymentUserData.mtdItId
   val sessionId: String = anEmploymentUserData.sessionId
   val affinityGroup: String = "affinityGroup"
-  val taxYear: Int = 2022
+  val taxYear: Int = DateTime.now().year().get()
```

Comment:
This is not enough. I think we need to add +1 as otherwise we will have the same problem after the 5th of April.